app/router.js

```diff
@@ -201,7 +201,7 @@ Router.map(function() {
     this.route('list');
     this.route('following');
   });
-  this.route('group-public', { path: '/g/:group_id' });
+  this.route('group-public', { path: '/groups/:group_id' });
```

Comment:
you have to change route on button not in router.

Reply:
@maze-runnar I tried that too but the button was not routing to groups so I changed the route to groups.

Reply:
We cant change route, you have to do it other way.